tox.ini

```
@@ -16,3 +16,23 @@ commands =
       --binary \
       --out-dir {toxinidir}/dist/ \
       {toxinidir}
+
+
+[testenv:lint]
+basepython = python3
+commands =
+    {envpython} -m pre_commit run --show-diff-on-failure {posargs:--all-files}
+
+    # Print out the advise of how to install pre-commit from this env into Git:
+    -{envpython} -c \
+    'cmd = "{envpython} -m pre_commit install"; \
+      scr_width = len(cmd) + 10; \
+      sep = "=" * scr_width; \
+      cmd_str = "    $ " + cmd;' \
+      'print("\n" + sep + "\nTo install pre-commit hooks into the Git repo, run:\n\n" + cmd_str + "\n\n" + sep + "\n")'
```

Since we already declare that it's py3-only, let's embrace f-strings here:
```suggestion
      cmd_str = "    $ \{cmd\}";' \
      'print(f"\n\{sep\}\nTo install pre-commit hooks into the Git repo, run:\n\n\{cmd_str\}\n\n\{sep\}\n")'
```